docs/concepts/conventions/simulation-experiments.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-Simulators should support SED-ML L1V3 or later. To accommodate a wide range of modeling frameworks and simulation algorithms, BioSimulators embraces the additional conventions for SED-ML described below, as well as the conventions for executing SED-ML documents described with the standard for [command-line interfaces for simulation tools](./simulator-interfaces.md).
+Simulators should support SED-ML L1V3 or later. To accommodate a wide range of modeling frameworks and simulation algorithms, BioSimulators embraces the additional conventions for SED-ML described below, as well as the conventions for executing SED-ML documents described with the standard that is used for [command-line interfaces of simulation tools](./simulator-interfaces.md).
```

Comment:
This refers to the standard interfaces themselves and should probably stay as it

Reply:
I see; disregard the edit :)